admin/app/models/complementary-certification-course-results-with-external.js

```diff
@@ -5,4 +5,8 @@ export default class ComplementaryCertificationCourseResultsWithExternal extends
   @attr('string') pixResult;
   @attr('string') externalResult;
   @attr('string') finalResult;
+
+  get isExternalResultEditable() {
+    return this.externalResult !== 'Rejetée' && this.externalResult !== '-';
```

Comment:
A voir avec @asrodride 
```suggestion
    return this.pixResult !== 'Rejetée';
```

Reply:
ok